OrbitGl/TopDownView.cpp

```diff
@@ -56,7 +79,7 @@ static void AddCallstackToTopDownThread(TopDownThread* thread_node,
                                         const CallStack& resolved_callstack,
                                         uint64_t callstack_sample_count,
                                         const CaptureData& capture_data) {
-  TopDownInternalNode* current_thread_or_function = thread_node;
+  TopDownNode* current_thread_or_function = thread_node;
```

Comment:
Could you add some words to the commit message explaining what's happening in the first commit? I've seen you've merged Node and InternalNode and thread_children_/function_children_ have been introduced. But it would be great to understand the overall rationale.

Reply:
Done.